mobile/src/actions/settings.js

```diff
@@ -76,8 +78,12 @@ export const registerDevice = () => async (dispatch, getState) => {
     dispatch(setClient(client))
     startReplication(dispatch, getState)
   }).catch(err => {
-    dispatch(wrongAddressError())
-    logException(err)
+    if (err.message === 'Missing states from current URL') {
```

Comment:
This seems a bit fragile. Where does the error come from? Is there maybe a better way to detect it?